roles/kafka_connect/tasks/health_check.yml

```diff
@@ -9,6 +9,8 @@
     oauth_password: "{{ kafka_connect_oauth_password }}"
     ldap_user: "{{ kafka_connect_ldap_user }}"
     ldap_password: "{{ kafka_connect_ldap_password }}"
+    mds_mtls_client_cert: "{{ kafka_connect_cert_path }}"
```

Comment:
the variable name is misleading, we are not sending mds certs here, we send component wise cert for each component. Please rename it to `mtls_client_cert`

Reply:
by mds_mtls_client_cert i meant cert of MDS's client for mtls. and not cert of MDS. But I get that this might be misleading name. `mtls_client_cert` makes sense. will change this